embedly/Dockerfile

```diff
@@ -1,8 +1,14 @@
 FROM python:2.7
 
 EXPOSE 7001
-WORKDIR /embedly
-COPY . /embedly
-RUN pip install -r /embedly/requirements.txt
+
+RUN groupadd --gid 10001 app && \
+    useradd --uid 10001 --gid 10001 --home /app --create-home app
+
+WORKDIR /app
+COPY . /app
+RUN pip install -r /app/requirements.txt
```

Comment:
I'm under the impression that caching goes better if you copy just requirements.txt into the container first, run pip install, and then do